pythonExecSource/src/main/templates/Dockerfile.tmpl

```diff
@@ -1,10 +1,10 @@
 # syntax=docker/dockerfile:1
 
```

Comment:
Update from Python 3.8 -> 3.12 base.  That complies with our official restrictions.